modules/nf-core/malt/run/tests/nextflow.config

```diff
@@ -1,13 +1,16 @@
 process {
+    resourceLimits = [
+        memory: '6.GB'
+    ]
```

Comment:
```suggestion
```
So this shouldn't be defined here, but in the profile that determines where these tests are run.